dags/pydantic/types.py

```diff
@@ -510,7 +510,7 @@ def conset(item_type: Type[T], *, min_items: int = None, max_items: int = None)
     # __args__ is needed to conform to typing generics api
     namespace = {'min_items': min_items, 'max_items': max_items, 'item_type': item_type, '__args__': [item_type]}
     # We use new_class to be able to deal with Generic types
-    return new_class('ConstrainedSetValue', (ConstrainedSet,), {}, lambda ns: ns.update(namespace))
+    return new_class('ConstrainedSetValue', (ConstrainedSet,), {}, lambda ns: ns._update(namespace))
```

Comment:
this file also needs a rollback ;)

Reply:
Did a git diff on the whole pydantic directory, there were two more ;) Fixed them all now